test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj.UseDotNetCore22

```diff
@@ -1,17 +1,16 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Why do we have 3 different csproj for for different dotnet version? You can specify multiple through adding <TargetFrameworks>netcoreapp2.0;netcoreapp2.1;netcoreapp2.2</TargetFrameworks>

Reply:
Talked offline. I can make a new PR to change this but it's a bit out of scope of this PR